docs/servo_power_monitor/src/index.ts

```diff
@@ -79,14 +79,13 @@ selectDUTSerialButton.addEventListener('click', async () => {
 });
 
 const form = document.getElementById('form') as HTMLFormElement;
+const input = document.getElementById('input') as HTMLInputElement;
```

Comment:
Please use more descriptive name (but since it affects other lines, you can proceed with this for now)
e.g. DutShellInput